### ISSUE TITLE
[bug] easyengine 4.2.0 test failure

### ISSUE THREAD

**Issue by chenrui333** [EasyEngine/easyengine#1536]
👋 trying to build the latest release, but run into some build issue. The error log is as below:

<details>
<summary>test failure</summary>

```
$ /usr/local/Cellar/easyengine/4.2.0/bin/ee cli info
PHP Fatal error:  Uncaught Error: Attempt to modify property "site_types" on null in phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/src/Site_Command.php:46
Stack trace:
#0 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/site-command.php(40): Site_Command::add_site_type('html', 'EE\\Site\\Type\\HT...')
#1 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/composer/autoload_real.php(71): require('phar:///usr/loc...')
#2 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/composer/autoload_real.php(61): composerRequire68fd9c1e2fa70691d169e3da541e9e7e('894ce33214346ba...', 'phar:///usr/loc...')
#3 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/autoload.php(7): ComposerAutoloaderInit68fd9c1e2fa70691d169e3da541e9e7e::getLoader()
#4 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/EE/Bootstrap/AutoloaderStep.php(42): require('phar:///usr/loc...')
#5 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/bootstrap.php(74): EE\Bootstrap\AutoloaderStep->process(Object(EE\Bootstrap\BootstrapState))
#6 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/init-ee.php(27): EE\bootstrap()
#7 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/boot-phar.php(8): include('phar:///usr/loc...')
#8 /usr/local/Cellar/easyengine/4.2.0/bin/ee(5): include('phar:///usr/loc...')
#9 {main}
  thrown in phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/src/Site_Command.php on line 46

Fatal error: Uncaught Error: Attempt to modify property "site_types" on null in phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/src/Site_Command.php:46
Stack trace:
#0 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/site-command.php(40): Site_Command::add_site_type('html', 'EE\\Site\\Type\\HT...')
#1 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/composer/autoload_real.php(71): require('phar:///usr/loc...')
#2 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/composer/autoload_real.php(61): composerRequire68fd9c1e2fa70691d169e3da541e9e7e('894ce33214346ba...', 'phar:///usr/loc...')
#3 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/autoload.php(7): ComposerAutoloaderInit68fd9c1e2fa70691d169e3da541e9e7e::getLoader()
#4 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/EE/Bootstrap/AutoloaderStep.php(42): require('phar:///usr/loc...')
#5 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/bootstrap.php(74): EE\Bootstrap\AutoloaderStep->process(Object(EE\Bootstrap\BootstrapState))
#6 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/init-ee.php(27): EE\bootstrap()
#7 phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/php/boot-phar.php(8): include('phar:///usr/loc...')
#8 /usr/local/Cellar/easyengine/4.2.0/bin/ee(5): include('phar:///usr/loc...')
#9 {main}
  thrown in phar:///usr/local/Cellar/easyengine/4.2.0/bin/ee/vendor/easyengine/site-command/src/Site_Command.php on line 46
```

</details>

full build log, https://github.com/Homebrew/homebrew-core/runs/2860590022
relates to Homebrew/homebrew-core#79579

**Comment by kirtangajjar** [comment]
@chenrui333 Thanks for reporting this. We are already aware of the issue, and we'll be fixing the issues as soon as possible :)

**Comment by chenrui333** [comment]
@kirtangajjar closing the issue now as it is fixed and now the formula is with php 8.0 per [4.3.0 update](https://github.com/Homebrew/homebrew-core/pull/80393)

**Comment by chenrui333** [comment]
Thanks!
